modules/countdown.js

```diff
@@ -3,46 +3,78 @@ require('moment-countdown')
 
 let timeout = null
 
-const ARTICLE_50 = '2020-01-31T00:00:00+01:00'
+function isPast (date) {
+  const now = moment()
+  const expiryDate = moment(date)
+  return expiryDate.isBefore(now)
+}
+
+function nextDate (milestones) {
+  for (const milestone in milestones) {
+    if (!isPast(milestone.date)) return milestone
+  }
+  return { name: null, date: null }
+}
+
+function nextBrexitDate () {
+  const milestones = [
+    { name: 'Article 50', date: '2020-01-31T00:00:00+01:00' },
+    { name: 'The transition period', date: '2019-12-31T00:00:00+01:00' }
+  ]
+  return nextDate(milestones)
+}
 
 function autoCount (bot, lastTick) {
-  const thisTick = moment(ARTICLE_50).countdown().toString().split(/, | and /)[0]
+  const { name, date } = nextBrexitDate
+  if (!name) return
+  const thisTick = moment(date).countdown().toString().split(/, | and /)[0]
   if (lastTick != null && thisTick !== lastTick) {
-    bot.broadcast('Article 50 expires in ' + lastTick)
+    bot.broadcast(`${name} expires in ${date}`)
   }
   timeout = setTimeout(autoCount, 1000, bot, thisTick)
 }
 
+function expiry (eventInfo) {
+  const { name, date } = eventInfo
+  const countdown = moment(date).countdown().toString()
+  if (isPast(date)) {
+    return `${name} expired ${countdown} ago`
+  }
+  return `${name} expires in ${countdown}`
+}
+
 module.exports = {
   commands: {
     a50: {
-      help: 'Gets the time until the UK Article 50 procedure expires',
+      help: "Gets the time until the next stage of the UK's withdrawal from the European Union",
       command: function () {
-        return 'Article 50 expires in ' + moment(ARTICLE_50).countdown().toString()
+        return expiry(nextBrexitDate)
```

Comment:
Do you mean to pass the `nextBrexitDate` function through, instead of the return value of the function?